include/deal.II/matrix_free/face_setup_internal.h

```diff
@@ -164,15 +164,14 @@ namespace internal
 
 
     template <int dim>
-    template <typename MFAddData>
     void
     FaceSetup<dim>::initialize(
-      const dealii::Triangulation<dim> &                  triangulation,
-      const MFAddData &                                   additional_data,
+      const dealii::Triangulation<dim> &triangulation,
+      const unsigned int                mg_level,
+      const bool                        hold_all_faces_to_owned_cells,
```

Comment:
That's good :) That `AdditionalData` is part of `MatrixFree` class is kind of annoying due to the template arguments...